Upload code verify Car For Sale Year Tab Component

anabelle2134 opened the pull request:
Create CarForSaleYearTabComponent class
Create CarForSaleYearTabComponentTest class

anabelle2134 committed:
Upload code verify Car For Sale Year Tab Component

Create CarForSaleYearTabComponent class
Create CarForSaleYearTabComponentTest class

namnguyen-itvn commented:
@doanhoangtri pls review this PR

doanhoangtri commented:
- Coding convention: fine
- test run OK
- Have not used Data Driven
- no Function Assert to verify TestCase

namnguyen-itvn commented on year/pom.xml:
```diff
@@ -0,0 +1,117 @@
+          <version>3.0.2</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-install-plugin</artifactId>
```
mmmm

doanhoangtri commented on year/src/test/java/com/example/test/pages/KBBMOOCProject/CarForSaleYearTabComponent.java:
```diff
@@ -0,0 +1,137 @@
+    }
+
+     //Declare element
+     WebElement tabYear = keyword.findElementByLocator(By.xpath("//span[contains(text(), 'Year')]"));
```
Not Modified

doanhoangtri replied:
add access modifier

doanhoangtri reviewed (commented):
Access Modifier

doanhoangtri reviewed (commented):
you need add access modifier

anabelle2134 committed:
Update code verify the year tab when user select max min year success
+ Verify all element is displayed as requirement
+ Verify title and url after user select maximum year and minimum year success

anabelle2134 committed:
no message

anabelle2134 committed:
commit change
